core/src/commonMain/kotlin/io/github/aag/core/di/Koin.kt

```diff
@@ -11,9 +23,35 @@ import org.koin.dsl.module
 fun initKoin(appDeclaration: KoinAppDeclaration = {}) =
     startKoin {
         appDeclaration()
-        modules(commonModule())
+        modules(
+            commonModule(),
+            networkModule()
+        )
     }
 
 fun commonModule() = module {
     singleOf(::ProfileRepositoryImpl) bind ProfileRepository::class
 }
+
+internal fun networkModule() = module {
+    singleOf<HttpClient> {
+        HttpClient {
+            install(Logging)
+            install(Resources)
+            install(ContentNegotiation) {
+                json()
```

Comment:
without any configuration?

Reply:
oops, it shouldn't return anything

Reply:
`json(Json { isLenient = true; ignoreUnknownKeys = true })`?